testing/e2e/skr/skr-binding-test/index.js

```diff
@@ -162,6 +162,32 @@ describe('SKR Binding test', function() {
     }
   });
 
+  it('Should not allow creation of more than 10 SKR bindings', async function() {
+    errorOccurred = false;
+    count = 0;
+    while (!errorOccurred && count < 13) {
+      bindingID = Math.random().toString(36).substring(2, 18);
```

Comment:
Could we replace it with uuid generation?

Reply:
No problem, I will replace it also in the existing parts of code.